react/src/App.tsx

```diff
@@ -85,11 +85,20 @@ function BaseApp(props: { darkMode: boolean; toggleDarkMode: () => void }) {
 
             const apiInfoResult = await fetch("/api");
             if (apiInfoResult.ok) {
-                const apiInfo = await apiInfoResult.json();
+                let apiInfo = await apiInfoResult.json();
+                if (!("endpoint" in apiInfo)) {
+                    apiInfo = { ...apiInfo, endpoint: null }; // null endpoint value -> use the default endpoint
+                }
```

Comment:
Are you supposing that the correct endpoint (domain or prefix) for this tenant will be in the response from the `/api` endpoint on the current domain?

Reply:
This was an assumption initially that @cklamann cleared up. That statement doesn't need to be wrapped in an if condition.

Reply:
No, the problem is that you can't make a request to the backend to figure out what backend to make requests to!

Reply:
Yeah, this code was initially done with the incorrect assumption. I'm gonna remove this for now as it'll be fixed in the second PR for this feature. This PR is for purely UI stuff